### DISCUSSION TITLE
How can k8sClient C# SDK print error logs when the renew-lease request get an error response.

### DISCUSSION THREAD

**Discussion by Moresoph** [kubernetes-client/csharp#1145] [Q&A]
We use k8s to elect a leader, the k8s will renew the lease every 2 seconds by the default configuration, But the problem is there are no error logs when the renew-lease request failed. So we want to do some configuration to show error logs when the renew-lease requests get an error response. How can we do that?

Here is our code:
![image](https://user-images.githubusercontent.com/13598265/208031275-ed5d68e3-2a0a-470d-bb3a-acb5fa202cf1.png)

**Comment by Moresoph** [comment]
Hi @tg123 could you give some suggestions about this question?

**Comment by tg123** [comment]
`onstopleading` works for you?
or you want detailed request err? those err is ignored atm

**Reply by tg123** [reply]
maybe you can send a PR to add onerror
